x-pack/solutions/observability/plugins/observability/server/lib/rules/custom_threshold/custom_threshold_executor.test.ts

```diff
@@ -1660,6 +1660,47 @@ describe('The custom threshold alert type', () => {
           },
         });
       });
+      test('includes reason message in the recovered alert context pulled from the last active alert ', async () => {
```

Comment:
I added the test only to the Custom Threshold as the goal is to deprecate/invest less in Logs, Metrics, and Inventory rules. 
Also, adding this test to other rules with the current structure/tools is time-consuming because it would require changing how the tests and the mock services work.